apps/et/et-cron-wa-task-for-expired-bfactions/prod.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: et-cron-wa-task-for-expired-bfactions
+spec:
+  releaseName: et-cron-wa-task-for-expired-bfactions
+  values:
+    job:
+      disableActiveClusterCheck: false
```

Just adding prod suspension as it's not live yet
```suggestion
    job:
      suspend: true
      disableActiveClusterCheck: false
```